mix/qml/js/ProjectModel.js

```diff
@@ -550,15 +585,10 @@ function registerToUrlHint()
 function normalizeAddress(addr)
 {
 	addr = addr.replace('0x', '');
-	var i = 0;
-	for (var k in addr)
-	{
-		if (addr[k] !== "0")
-			break;
-		else
-			i++;
-	}
-	return addr.substring(i);
+	if (addr.length <= 40)
+		return addr;
+	var left = addr.length - 40;
+	return addr.substring(left);
 }
 
 function formatAppUrl(url)
```

Comment:
This function returns string instead of array if passed an empty, or single-word path